### PR TITLE
bread crumbs

### PR DESCRIPTION
**Pull request by HeySleare**

### CODE REVIEW

File: src/components/BreadCrumbs/BreadCrumbs.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import './BreadCrumbs.scss';
+
+export const BreadCrumbs: React.FC = () => {
+  const location = useLocation();
+
+  let currentLink = '';
+  const crumbs = location.pathname.split('/')
+    .filter(crumb => crumb !== '' && !+crumb)
+    .map(crumb => {
+      currentLink += `/${crumb}`;
```
Comment (artem8746) [inline src/components/BreadCrumbs/BreadCrumbs.tsx:11]: move it to function, would be better to use const instead

File: src/components/BreadCrumbs/BreadCrumbs.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        <>
+          <div className="bread-crumbs__crumb bread-crumbs__arrow" />
+          <div className='bread-crumbs__crumb' key={crumb}>
+            {/* {crumb!=='catalog' && ( */}
+            <Link className='bread-crumbs__link' to={currentLink}>{crumb}</Link>
+            {/* )} */}
+          </div>
+        </>
+      );
+    });
```
Comment (artem8746) [inline src/components/BreadCrumbs/BreadCrumbs.tsx:23]: move it to function and put to services, use const instead let

**Review by artem8746 — CHANGES_REQUESTED** [review]
💦💦
( •_•)>⌐■-■

**Review by sasholk — APPROVED** [review]
🦆

**Pull request merged**